websocketOutportDriver/payloadParser.go

```diff
@@ -102,3 +105,49 @@ func padUint32ByteSlice(initial []byte) []byte {
 	padding := bytes.Repeat([]byte{0}, 4)
 	return append(padding, initial...)
 }
+
+func prepareArgsSaveBlock(args *outport.ArgsSaveBlockData) *outport.ArgsSaveBlockData {
```

Comment:
Please make the args param as value as you did not check it for nil

Reply:
added an extra check

Reply:
refactored